src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -51,23 +108,23 @@ var Marathon = React.createClass({
 
     Mousetrap.bind("esc", function () {
       if (this.refs.modal != null) {
-        this.refs.modal.destroy();
+        this.modalDestroy();
       }
     }.bind(this));
 
     Mousetrap.bind("c", function () {
```

Comment:
Can we just remove this binding? It gets in the way of copy/paste and is a usability issue. Happy to handle it in another PR.